pkg/app/server/option.go

```diff
@@ -151,6 +152,16 @@ func WithHostPorts(hp string) config.Option {
 	}}
 }
 
+// WithBasePath sets basePath
+func WithBasePath(basePath string) config.Option {
```

Comment:
Unit tests need to be added.

Reply:
ok

Reply:
I think the code is OK, but I don't see a joint test with Hertz server in the unit test section, only a unit test for option. Similar to [it ](https://github.com/cloudwego/hertz/blob/develop/pkg/app/server/hertz_test.go#L307)